compact/far_backward/lumi/collimator.xml

```diff
@@ -0,0 +1,28 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Justin Chan -->
+
+<lccdd>
+
+  <comment>
+  Collimater outer dimensions chosen to match the ZEUS design for the Luminosity monitor.
+  Inner dimensions chosen to be 5*sigma for photon beam evaluated at far Z-end of collimator location, calculated using the max angular divergencies of the electron beam.
+  Outer dimensions are full lengths.  Inner ones are half lengths.
```

Comment:
This seems like it will cause confusion later. This should be avoided, or be clear in the variables/attributes names, like `dz` for half.